libs/core-ui/src/lib/Cohort/ManualCohortManagement/CohortName.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Link, Stack, Text } from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { Cohort } from "../Cohort";
+
+export interface ICohortNameProps {
+  cohort: Cohort;
+  index: number;
+  cloneAndEditCohort(index: number): void;
+  editCohort(index: number): void;
+}
+
+export class CohortName extends React.Component<ICohortNameProps> {
+  public render(): React.ReactNode {
+    return (
+      <Stack>
+        <Text>{this.props.cohort.name}</Text>
+        <Stack horizontal tokens={{ childrenGap: "s1" }}>
+          {this.props.index && (
+            <Link onClick={this.onClickEditCohort}>
+              {localization.Interpret.CohortBanner.editCohort}
```

Comment:
This localization path is weird.  It should be in core-ui.  I guess this goes with issue https://github.com/microsoft/responsible-ai-toolbox/issues/1340

Reply:
Yeah good point, we can update the localization path when we are working on issue https://github.com/microsoft/responsible-ai-toolbox/issues/1340